### PR TITLE
make default target now shows list of available make commands

### PR DESCRIPTION
**Pull request by uweschmitt**
I took the solution from https://stackoverflow.com/questions/4219255/how-do-you-get-the-list-of-targets-in-a-makefile/#answer-26339924

```bash
$ make
Call a specific subcommand:

build
clean
debug
deps
docs
initdb
licenses
lint
list
purge
reformat
reindex
serve
shell
stop
tests
travis-deps

make: *** [default] Error 1
```

### CODE REVIEW

**Comment by nlhkabu** [comment]
Thanks for this PR @uweschmitt :)

@di @ewdurbin our second PR from the europython sprints! Can you please review?

File: Makefile [hunk 1]
```diff
@@ -167,4 +173,4 @@ purge: stop clean
 stop:
 	docker-compose down -v
 
-.PHONY: default build serve initdb shell tests docs deps travis-deps clean purge debug stop
+.PHONY: default build serve initdb shell tests docs deps travis-deps clean purge debug stop list
```
Comment (di) [inline Makefile]: Did you mean to include this line?

Comment (uweschmitt) [reply Makefile]: Was not meant to be included, f5d7e08 fixes this.

Comment (di) [reply Makefile]: Looks like it got re-added in 8c8349f though.

Comment (uweschmitt) [reply Makefile]: should be fixed now.

**Comment by di** [comment]
Thanks @uweschmitt!

**Pull request merged**